.github/workflows/ci_submodule.yml

```diff
@@ -0,0 +1,69 @@
+name: CI for libsc check
+on:
+  pull_request:
+
+jobs:
+
+  linux-sub:
+    permissions: write-all
+    if: |
+      github.event.pull_request.draft == false &&
+      github.event.pull_request.author_association != 'COLLABORATOR'
+    runs-on: ubuntu-latest
+    env:
+      TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      PR_NODE_ID: ${{ github.event.pull_request.node_id }}
+      INPUT_PATH: sc
+    steps:
+    - name: Checkout source code for base
+      uses: actions/checkout@v4
+      with:
+        submodules: false
+        ref: ${{ github.event.pull_request.base.ref }}
+        path: base
+
+    - name: Checkout source code for head
+      uses: actions/checkout@v4
+      with:
+        repository: ${{ github.event.pull_request.head.repo.full_name }}
+        ref: ${{ github.event.pull_request.head.ref }}
+        submodules: false
+        path: head
+        
+    - name: Determine hashes to compare
+      shell: bash
+      run: |
+        EVENT_PATH="${GITHUB_EVENT_PATH}"
+        cd "${GITHUB_WORKSPACE}" \
```

Comment:
One more comment: do we need this line? It seems we are already in this directory.

Reply:
You are right, we started in the same solder. It's fixed now.